test/distributed/_composable/fsdp/test_fully_shard_autograd.py

```diff
@@ -326,5 +328,90 @@ def optim_hook(param: nn.Parameter) -> None:
                 self.assertTrue(torch.equal(ref_param, param))
 
 
+class TestFullyShardGradientScaler(FSDPTest):
```

Comment:
Shall I put the test here?

Reply:
Looking through the test files, I think this is okay, but cc @weifengpy in case he has better opinions.